apache/files/FreeBSD/apache-2.4.config.jinja

```diff
@@ -526,5 +526,9 @@ TraceEnable Off
 # Well, IncludeOptional behaved lile Include
 IncludeOptional etc/apache24/extra/security.con[f]
 
+{% if salt['pillar.get']('apache:mod_ssl:manage_tls_defaults', False) -%}
+Include etc/apache24/extra/tls-defaults.conf
```

Comment:
I don't have any FreeBSD around (so I don't know if these paths should be relative or absolute), but isn't this line missing an initial '/' in the `Include` path?

Reply:
looks like the configuration from:
http://www.xfiles.dk/guide-on-how-to-configure-the-apache-24-httpd-conf-file-on-freebsd/

Reply:
Yeah, reading the ` apache/map.jinja`, it seems all these files are relative to the `/usr/local` dir.